### DISCUSSION TITLE
Immich does not work after system restart

### DISCUSSION THREAD

**Discussion by maximgvozdev** [immich-app/immich#13483] [Q&A]
Hi All!

I have last clear Ubuntu, install last docker and install clear last Immich.

I uploaded some photos. Everything works.
All I changed is:
UPLOAD_LOCATION=/media/user/photo-ssd/immich

(photo-ssd it's SSD)

After restart my PC i see new folder (or drive, Excuse me, Ubuntu is a dark forest for me) "photo-ssd1". In new folder i see Immich data... immich docker started, but don't work...

If i stop docker, change UPLOAD_LOCATION to /media/user/photo-ssd1/immich and start docker - all ok... until I reboot the PC... 
I see new folder "photo-ssd2"... and so on ad infinitum... what could be the problem?

**Comment by bo0tzz** [comment]
Please provide more detail: Your compose file, logs from the containers, etc.

**Comment by maximgvozdev** [comment]
As I understand it, by default the disk (in my case SATA) is mounted after logon. (WHY????)
It helped me

https://askubuntu.com/questions/164926/how-to-make-partitions-mount-at-startup
